ddl/ddl_api.go

```diff
@@ -562,6 +566,19 @@ func columnDefToCol(ctx sessionctx.Context, offset int, colDef *ast.ColumnDef, o
 		Version: model.CurrLatestColumnInfoVersion,
 	})
 
+	if col.FieldType.Tp == mysql.TypeBlob {
+		if col.FieldType.Flen <= tinyBlobMaxLength {
+			logutil.BgLogger().Info("Auto convert Blob to TinyBlob", zap.Int("Blob Length", col.FieldType.Flen))
```

Comment:
```suggestion
			logutil.BgLogger().Info(fmt.Sprintf("Automatically convert BLOB(%d) to TINYBLOB", col.FieldType.Flen))
```
How about this?

IMHO it's better to use all upper-cased word('TINYBLOB', 'MEDIUMBLOB' ...) compared with the camel styled ones because looks this is what MySQL is documented.